### ISSUE TITLE
accessible breadcrumb navigation

### ISSUE THREAD

**Issue by sonusharma6-dsa** [SAPTARSHI-coder/EaseMotion-css#4961]
## Feature Request

Breadcrumb navigation is missing from EaseMotion. Should use semantic `<ol>` markup with `aria-label` on the `<nav>` and `aria-current="page"` on the active item. Separator should be customisable via a CSS variable.

```html
<nav aria-label="Breadcrumb">
  <ol class="ease-breadcrumb">
    <li><a href="/">Home</a></li>
    <li><a href="/docs">Docs</a></li>
    <li aria-current="page">Getting Started</li>
  </ol>
</nav>
```

**Comment by sonusharma6-dsa** [comment]
This is something I have been meaning to address. Please assign me to this one.
